docs/development/virtual_environments.rst

```diff
@@ -117,8 +117,12 @@ If you want to receive OSSEC alerts or change any other settings, you will need
 to fill out your local copy of
 ``./install_files/ansible-base/staging-specific.yml``.
 
+You should first provision the VM required for building the app code
+Debian packages on the staging machines:
+
 .. code:: sh
 
+   vagrant up build
```

Comment:
So... in practice I actually use `vagrant up --no-provision build`, since running `vagrant up /staging/` will also provision the `build` machine. Maybe that's overly technical, but in dev docs, it strikes me as honest.